src/js/plugins/stellar/plugin.js

```diff
@@ -134,6 +144,11 @@ const transformTransaction = (path, transaction) => {
             operation.signer = transformSigner(operation.signer);
         }
 
+        // transform asset path
+        if (operation.path) {
+          operation.path = transformPath(operation.path);
```

Comment:
wouldn't be better to write `operation.path = operation.path.map(transformAsset)` instead of creating a new function?

Reply:
Absolutely.